### ISSUE TITLE
If use GT-img for training?

### ISSUE THREAD

**Issue by renliao** [XiangZ-0/GEM#11]
Hello, thank you for your outstanding work. I noticed that this is a self supervised method that is very suitable for real-world scenarios. A total of three datasets were used for evaluation, with Ev REDS and HS-ERGB having GT images and MS-RBD not having GT images. My understanding is that during the training process, GT was not used as supervision and was only used to evaluate the final performance of the model. Is my understanding correct?

**Comment by XiangZ-0** [comment]
Yes, your understanding is correct! Our experiments are designed to imitate real-world scenarios, so all the GEM models are trained in a self-supervised manner without GT. GT is only used to compute metrics in our paper.

**Comment by renliao** [comment]
Thank you for your replay, I understand your work better now.
